src/contracts.js

```diff
@@ -54,13 +54,13 @@
     };
     BlameObj.prototype.addLocation = function(loc) {
         return Blame.clone(this, {
-            loc: this.loc != null ? this.loc.concat(loc) : [loc]
+            loc: this.loc !== null ? this.loc.concat(loc) : [loc]
```

Comment:
Can't actually do this because I was relying on the `null`/`undefined` checking of `==`/`!=`. In this case `loc` should never be `null` however so you could just switch it to `!== undefined`.